tests/integration/api_v3/rpc/generators.test.js

```diff
@@ -69,12 +69,12 @@ describe('Generators API', () => {
 			expect(result.data.length).toBeLessThanOrEqual(103);
 			result.data.map(generator => expect(generator).toMap(generatorSchema));
 
-			const activeGenerators = result.data
-				.filter(generator => generator.status === STATUS.ACTIVE);
-			const standbyGenerators = result.data
-				.filter(generator => generator.status === STATUS.STANDBY);
-			expect(activeGenerators.length).toEqual(numberActiveValidators);
-			expect(standbyGenerators.length).toEqual(numberStandbyValidators);
+			// const activeGenerators = result.data
```

Comment:
Add it to the acceptance criteria in https://github.com/LiskHQ/lisk-service/issues/1491.